mcc/Dispatcher.c

```diff
@@ -607,14 +607,14 @@ static IPTR mDraw(struct IClass *cl, Object *obj, struct MUIP_Draw *msg)
     ScrollIntoDisplay(data);
     data->update = TRUE;
 */
-
+/*
```

Comment:
Please remove all commented out code in /* */ if it is really not required anymore. That's exactly what a revision control system is all about.